notebooks/byod.py

```diff
@@ -98,30 +98,41 @@ def upload_columns(table: str, columns: Dict[str, List[Any]]):
             tsv_data += os.linesep + "\t".join([f"{i}", *[columns[h][i] for h in column_headers]])
         upload_data_table(tsv_data)
 
+    def parse_cram_crai(filename: str):
+        sample, ext = filename.rsplit(".", 1)
+        if sample.endswith("cram"):
```

Comment:
So... what if someone has a file named: `oh.my_silly_cram.cram`?

This function should probably, explicitly, check and remove what it's expecting.

Reply:
This was marked resolved, but was I see no commit afterwards?

Reply:
Because the fix was just a one character change. It'll be included in the commit that includes defaultdict.